### PR TITLE
Fix/graphql nextjs missing deps Fixes #4985

### LINKED ISSUE #4985: GraphQL Next.js example missing dependencies
Trying to build throws following error

```
Type error: Cannot find module 'graphql-tag' or its corresponding type declarations.
```

### PR DESCRIPTION
**Pull request by mayank1513**
Fixes #4985 4985

### CODE REVIEW

File: typescript/graphql-nextjs/package.json [hunk 1]
```diff
@@ -33,5 +34,10 @@
   },
   "prisma": {
     "seed": "ts-node prisma/seed.ts"
+  },
+  "pkg": {
+    "assets": [
+      "node_modules/.prisma/client/*.node"
+    ]
```
Comment (ruheni) [inline typescript/graphql-nextjs/package.json]: Could you clarify what this snippet does & how it impacts this example?

Comment (mayank1513) [reply typescript/graphql-nextjs/package.json]: Found the solution here - https://www.prisma.io/docs/guides/other/troubleshooting-orm/help-articles/pkg-issue

Comment (mayank1513) [reply typescript/graphql-nextjs/package.json]: Anyone actively maintains this repo?

I can volunteer!

**Comment by jharrell** [comment]
Closing as it looks like this is a noop now. Thank you for the PR, though!

**Pull request closed without merging**